orchestrator_mapreduce.py

```
@@ -386,66 +894,28 @@ async def _retry_subtask(self, subtask: SubTask):
-        """Persist job state to STATE.md for cross-run continuity."""
-        state_entry = (
-            f"\n## Job: {job.id}\n"
-            f"- **Intent:** {job.intent}\n"
-            f"- **Status:** {job.status.value}\n"
-            f"- **Verification:** "
-            f"{'PASSED' if job.verification_passed else 'FAILED'}\n"
-            f"- **Subtasks:** {len(job.subtasks)} total, "
-            f"{sum(1 for st in job.subtasks if st.status == TaskStatus.COMPLETED)} completed\n"
-            f"- **Timestamp:** {datetime.now(timezone.utc).isoformat()}\n"
-        )
+    async def _persist_state(self, job: OrchestratedJob):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_persist_state`

Add return type annotation: `None`

(ANN202)